Library/Homebrew/test/formula_pin_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "formula_pin"
+
+describe FormulaPin do
+  subject { described_class.new(formula) }
+  let(:formula) { formula_double_class.new }
```

Comment:
I believe it's something like `allow(the_double).to receive(:installed_prefixes) { … }`